Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -734,15 +791,29 @@ protected class ClassWriter : IClassWriter {
       convertE1_to_int = false;
 
       switch (op) {
+        case BinaryExpr.ResolvedOpcode.And:
+          opString = "and"; break;
+
         case BinaryExpr.ResolvedOpcode.Add:
+        case BinaryExpr.ResolvedOpcode.Concat:
           opString = "+"; break;
 
         case BinaryExpr.ResolvedOpcode.Mul:
           opString = "*"; break;
 
         case BinaryExpr.ResolvedOpcode.EqCommon:
+        case BinaryExpr.ResolvedOpcode.SeqEq:
+        case BinaryExpr.ResolvedOpcode.SetEq:
+        case BinaryExpr.ResolvedOpcode.MapEq:
           opString = "=="; break;
 
+        case BinaryExpr.ResolvedOpcode.Union:
+          opString = "|"; break;
+
+        case BinaryExpr.ResolvedOpcode.InSet:
```

Comment:
This will work for `InSeq` and `InMap` too, right? I assume it's just not used the tests so far?